### DISCUSSION TITLE
Ddt4all source problem

### DISCUSSION THREAD

**Discussion by Eneshafiz** [cedricp/ddt4all#1121] [General]
Hi all, after installing the program and moving the source zip az “ecu” to the program folder, the program refused to enter the edition mode or make any connection with the obd adapter. What could be the problem? 
I tried to install different versions but still have the same issue

**Comment by Furtif** [comment]
put the zip in the root of the ddt4all folder, do not unzip it https://github.com/cedricp/ddt4all/discussions/845

**Reply by Eneshafiz** [reply]
I tried to do both ways (with zip and unzip) but same result. 
Any other suggestions?

**Reply by Furtif** [reply]
you should never unzip, you should put ecu.zip like this.

read https://github.com/cedricp/ddt4all/discussions/1098

**Reply by Eneshafiz** [reply]
I tried it also but same result unfortunately 😕

**Reply by Eneshafiz** [reply]
Any other way to try?

**Reply by Eneshafiz** [reply]
Need help please 🙏🏻

**Reply by Furtif** [reply]
python -m pip install -r ./requirements.txt
python ./main.py

**Reply by Eneshafiz** [reply]
Can you be more specific please?
